template-parts/flexible-cpts/listing-filters.php

```diff
@@ -9,137 +9,33 @@
     foreach ($listing_filters as $filter) {
 
-        $taxonomy = get_taxonomy($filter);
-
-        if (!$taxonomy) {
-            continue;
-        }
-
-        $parent_class_name = str_replace(' ', '-', $taxonomy->name . '-filter-topic');
-        $child_class_name = str_replace(' ', '-', $taxonomy->name . '-filter-subtopic');
-
-        // Get the selected parent topic
-        $selected_topic = get_query_var($taxonomy->query_var);
-
-        // Use a unique query var for each genre_subtopic
-        $subtopic_query_var = $taxonomy->query_var . '_subtopic';
-        $selected_sub_topic = get_query_var($subtopic_query_var);
-
-        // Construct the field name for the restriction based on the filter
-        $restrict_field = 'restrict_by_' . $filter;
-
-        // ACF 'restrict_by_*' custom field is generated via code
-        // https://github.com/ministryofjustice/hale/blob/6d5ca3c9c6ddbcf27b23857223a54bcdf5778def/inc/flexible-cpts.php
-        $restrict_terms = get_field($restrict_field);
-
-        // todo: check if tax slug is restricted
-        //$taxonomy_term_ids = get_taxonomy_term_ids($restrict_taxonomies_array);
-
-        if (empty($restrict_terms)) {
-            $dropdown_exclude = "";
+        if (taxonomy_exists($filter)) {
+            get_template_part('template-parts/flexible-cpts/taxonomy-filter', false, ['taxonomy-name' => $filter ]);
         }
-
-        if (is_array($restrict_terms) && !empty($restrict_terms)) {
-            // Get an array of terms excluding the restricted ones
-            $exclude_terms = get_terms([
-                'taxonomy' => $filter,
-                'exclude' => $restrict_terms
-            ]);
-
-            if (!empty($exclude_terms)) {
-                // For all the terms left after excluding the restricted ones
-                // get their ids into an array
-                $dropdown_exclude = array_map(function($term) {
-                    return $term->term_id;
-                }, $exclude_terms);
-            }
-        }
-
-        $dropdown_args = [
-            "name" => $taxonomy->query_var,
-            "id" => $parent_class_name,
-            "class" => "govuk-select",
-            'taxonomy' => $filter,
-            'show_option_all' => "Select option",
-            'depth' => 1,
-            'orderby' => 'name',
-            'order' => 'ASC',
-            'hierarchical' => 1,
-            'selected' => $selected_topic,
-            'exclude' => $dropdown_exclude
-        ];
-
-        $filter_label = $taxonomy->labels->singular_name;
-
-        if(isset($taxonomy->labels->listing_page_filter) && !empty($taxonomy->labels->listing_page_filter)){
-            $filter_label = $taxonomy->labels->listing_page_filter;
-        }
-
-        if($filter == "category"){
-            $filter_label = "Topic";
+        else if($filter == 'published-date'){
+            get_template_part('template-parts/flexible-cpts/date-filter', false, ['name' => "date_published"]);
```

Comment:
```suggestion
            get_template_part('template-parts/flexible-cpts/date-filter', false, ['name' => "date_published", 'label' => "Published date"]);
```

Reply:
This should be 'label' => "" unless we want to chaneg the label

Reply:
fixed